go/host/db/rollups.go

```diff
@@ -116,3 +120,26 @@ func (db *DB) writeTipRollupHeader(w ethdb.KeyValueWriter, val gethcommon.Hash)
 	}
 	return nil
 }
+
+// Stores if a rollup is in a block
+func (db *DB) writeRollupByBlockHash(w ethdb.KeyValueWriter, header *common.RollupHeader) error {
+	// Write the encoded header
+	data, err := rlp.EncodeToBytes(header)
+	if err != nil {
+		return err
+	}
+	key := rollupBlockKey(header.L1Proof)
+	fmt.Println("stored rollup ", header.Hash(), " for block ", header.L1ProofNumber)
```

Comment:
Another lingering print, worth a quick search for them

Reply:
Should be fixed now :)